src/test/routes/hooks.ts

```diff
@@ -1,11 +1,18 @@
 import { expect } from 'chai'
+import * as express from 'express'
 import * as mock from 'nock'
 
 import * as idamServiceMock from '../http-mocks/idam'
 
-export function attachDefaultHooks () {
+export function attachDefaultHooks (app?: express.Express) {
   let retrieveServiceTokenMock: any
 
+  before(() => {
+    if (app) {
+      app.locals.csrf = 'dummy-token'
```

Comment:
Significant amount of route tests started failing after making rendering more strict due to the fact that CSRF token wasn't there. Enabling CSRF module for tests is still not convenient for tests so this sets dummy token instead just to make views render.